src/core/Item.java

```diff
@@ -0,0 +1,116 @@
+package core;
+
+import commands.Command;
+
+import java.io.File;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ */
+public class Item {
+
+    private String name;
+    private String location;
+    private String type;
+    private String fileSize;
+    private Date lastModified;
+    private String filePermissions;
+    private Map<String, String> map;
+
+    public Item (File fileSystemItem) {
+        map = new HashMap<String, String>();
+        lastModified = new Date(fileSystemItem.lastModified());
+        location = fileSystemItem.getPath();
+
+        configureNameAndType(fileSystemItem);
+
+        configureSize(fileSystemItem);
+
+        configurePermissions(fileSystemItem);
+    }
+
+    private void configurePermissions(File fileSystemItem) {
+        filePermissions = "";
+        if (fileSystemItem.canRead()) {
```

Comment:
to option the permissions in this format please check this functionality PosixFileAttributeView